frontend/src/components/RejectionReason/Reasons/TokenModuleReject.vue

```diff
@@ -35,18 +25,8 @@
 					Required balance :
 					<b>
 						{{
-							// Todo: fix this when we reset the devnet DB
-							reason.details[reason.reasonType]?.requiredBalance?.value /
-							10 **
-								reason.details[reason.reasonType]?.requiredBalance?.decimals[
-									'$serde_json::private::Number'
-								]
-								? reason.details[reason.reasonType]?.requiredBalance?.value /
-								  10 **
-										reason.details[reason.reasonType]?.requiredBalance
-											?.decimals['$serde_json::private::Number']
-								: Number(reason.details.required_balance.value) /
-								  10 ** Number(reason.details.required_balance.decimals)
+							Number(reason.details.required_balance.value) /
```

Comment:
Bad idea to parse the value into `Number`.

We should have functions for that so we don't repeat the same math formula over and over again and I would like to get away from too much parsing of data types when you write your front-end code.